tensorflow/python/ops/array_ops.py

```diff
@@ -1630,9 +1636,21 @@ def boolean_mask_v2(tensor, mask, axis=None, name="boolean_mask"):
   ragged tensors, and can be used if you need to preserve the masked dimensions
   of `tensor` (rather than flattening them, as `tf.boolean_mask` does).
 
+  Examples:
+
+  >>> tensor = [0, 1, 2, 3]  # 1-D example
+  >>> mask = np.array([True, False, True, False])
+  >>> boolean_mask(tensor, mask)
```

Comment:
```>>> boolean_mask(tensor, mask)```
should be 
```>>> tf.boolean_mask(tensor, mask)```
Do the same to all parts where you missed calling ```tf.``` before the function names